core/iwasm/interpreter/wasm.h

```diff
@@ -1079,6 +1079,8 @@ struct WASMModule {
 
     /* Whether the underlying wasm binary buffer can be freed */
     bool is_binary_freeable;
+
+    uint32_t package_version;
```

Comment:
Same as above, how about changing to `uint32 package_version` and moving it to be after `uint32 module_type`?